### ISSUE TITLE
Nav Unification: JavaScript error in block editor, post-checkout

### ISSUE THREAD

**Issue by jeherve** [Automattic/jetpack#18493]
### Steps to reproduce the behavior

1. Start with a free WordPress.com site, in the block editor.
2. Insert a paid block, which will display an upgrade nudge.
3. Click the nudge and be redirected to the checkout page in Calypso, with the plan you're buying.
4. Upon purchase, you're redirected back to the editor where you can now use the block.
5. Notice this JS error in the console:

```js
Uncaught TypeError: Cannot read property 'querySelectorAll' of null
    at setAriaExpanded (admin-menu.js?m=1607965432h&ver=wpcom:7)
```
-- https://github.com/Automattic/jetpack/blob/e13487d0dbbe4c743739b6f1a3e37c143a8c2614/projects/plugins/jetpack/modules/masterbar/admin-menu/admin-menu.js#L7

******

cc @obenland

**Comment by jeherve** [comment]
Also reported in p8oabR-BT-p2#comment-4952

**Comment by getdave** [comment]
Will be fixed in JP 9.6
